modules/ROOT/pages/import-api-specification-exchange.adoc

```diff
@@ -57,6 +57,15 @@ Manage the API specifications linked to Exchange in your project from the API sp
 [NOTE]
 If you want Studio to list your API when searching in Exchange, you must mark your API specification published as an asset as *stable*.
 
+== Rescaffold an API Specification in a Mule Project
+
+After changing your API spec, such as adding a new endpoint, re-scaffold the spec to update your Mule project.
+
+. Right-click on the file.xml of your Mule project.
+. Select *API Specs*, the name of your Mule project (filename.xml), and *Generate Flows*.
```

Comment:
filename.xml or file.xml?

Reply:
Is Generate Flows a button? Should it be ", and then click Generate Flows"?

Reply:
"Generate Flows" if one of the options in the context menu when you right click.